CODEOWNERS

```diff
@@ -5,22 +5,22 @@
 # For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
 
 # These are the default owners for the whole content of the `control-plane` repository. The default owners are automatically added as reviewers when you open a pull request, unless different owners are specified in the file.
-* @PK85 @akgalwas @franpog859 @ralikio @mvshao @VOID404 @koala7659 @piotrmiskiewicz @szwedm @wozniakjan @ukff @MarekMichali @jaroslaw-pieszka
+* @akgalwas @franpog859 @mvshao @VOID404 @koala7659 @kyma-project/gopher
```

Comment:
@PK85 can you please ensure https://github.com/orgs/kyma-project/teams/gopher has write access to https://github.com/kyma-project/control-plane?